superset/connectors/druid/models.py

```diff
@@ -859,6 +862,12 @@ def values_for_column(self,
             'Getting values for columns [{}] limited to [{}]'
             .format(column_name, limit))
         # TODO: Use Lexicographic TopNMetricSpec once supported by PyDruid
+        dim_qry = column_name
+        if self.get_column(column_name).dimension_spec and\
```

Comment:
let's call `self.get_column(column_name)` only once: `col = self.get_column(column_name)`

Reply:
indent isn't right
```python
if (
        self.get_column(column_name).dimension_spec and
        ...
):
    ...
```

Reply:
I fix indentation as per flake8 but it looks weird...